libs/tracing/build.gradle

```
@@ -0,0 +1,47 @@
+plugins {
+    id 'corda.common-publishing'
+    id 'corda.common-library'
+}
+description 'Tracing'
+
+ext {
+    releasable = true
+}
+
+dependencies {
+    compileOnly 'org.osgi:osgi.annotation'
+    compileOnly 'org.osgi:osgi.core'
+    compileOnly 'org.osgi:org.osgi.service.component.annotations'
+
+    implementation platform("net.corda:corda-api:$cordaApiVersion")
+    api 'net.corda:corda-base'
+    api 'net.corda:corda-application'
+    implementation 'net.corda:corda-config-schema'
+    implementation 'org.jetbrains.kotlin:kotlin-osgi-bundle'
+    implementation "org.slf4j:slf4j-api:$slf4jVersion"
+
+    implementation project(":libs:configuration:configuration-core")
+
+    implementation("io.zipkin.reporter2:zipkin-sender-urlconnection:2.16.3")
```

Probably would make sense to introduce `zipkinVersion=2.16.3` and then:
```suggestion
    implementation("io.zipkin.reporter2:zipkin-sender-urlconnection:$zipkinVersion")
```